CMake version issue with `pip install`

On Mac, attempting to install throws a CMake version issue, related to this line in the source: https://github.com/njzjz/openbabel/blob/77993b9a3b96fb9bd86249098beb97ab0fcbafc6/CMakeLists.txt#L3

Presumably, this means either `scitkit-build` needs to be configured to use an older CMake or the `CMakeLists.txt` needs to be updated

```console
$ pip install openbabel-wheel
Collecting openbabel-wheel
  Using cached openbabel-wheel-3.1.1.21.tar.gz (13 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: openbabel-wheel
  Building wheel for openbabel-wheel (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for openbabel-wheel (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [38 lines of output]
      /private/var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/pip-build-env-8zwn4f2l/normal/lib/python3.9/site-packages/setuptools_scm/git.py:310: UserWarning: git archive did not support describe output
        warnings.warn("git archive did not support describe output")
      /private/var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/pip-build-env-8zwn4f2l/normal/lib/python3.9/site-packages/setuptools_scm/git.py:328: UserWarning: unprocessed git archival found (no export subst applied)
        warnings.warn("unprocessed git archival found (no export subst applied)")
      *** scikit-build-core 0.11.2 using CMake 4.0.2 (wheel)
      *** Configuring CMake...
      loading initial cache file /var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/tmprqgh0c7e/build/CMakeInit.txt
      -- The CXX compiler identification is AppleClang 14.0.3.14030022
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/g++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Found Python: /Users/Mark/Documents/UW/Research/Development/venv/bin/python (found version "3.9.1") found components: Interpreter
      -- EIGEN3_INCLUDE_DIR=/private/var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/pip-build-env-8zwn4f2l/overlay/lib/python3.9/site-packages/cmeel.prefix/include/eigen3
      CMake Warning (dev) at /private/var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/pip-build-env-8zwn4f2l/normal/lib/python3.9/site-packages/cmake/data/share/cmake-4.0/Modules/FetchContent.cmake:1953 (message):
        Calling FetchContent_Populate(openbabel) is deprecated, call
        FetchContent_MakeAvailable(openbabel) instead.  Policy CMP0169 can be set
        to OLD to allow FetchContent_Populate(openbabel) to be called directly for
        now, but the ability to call it with declared details will be removed
        completely in a future version.
      Call Stack (most recent call first):
        CMakeLists.txt:19 (FetchContent_Populate)
      This warning is for project developers.  Use -Wno-dev to suppress it.
      
      CMake Error at /var/folders/9t/tqc70b7d61v753jkdbjkvd640000gp/T/tmprqgh0c7e/build/_deps/openbabel-src/CMakeLists.txt:3 (cmake_minimum_required):
        Compatibility with CMake < 3.5 has been removed from CMake.
      
        Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
        to tell CMake that the project requires at least <min> but has been updated
        to work with policies introduced by <max> or earlier.
      
        Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
      
      
      -- Configuring incomplete, errors occurred!
      
      *** CMake configuration failed
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for openbabel-wheel
Failed to build openbabel-wheel
ERROR: Could not build wheels for openbabel-wheel, which is required to install pyproject.toml-based projects
```

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -140,6 +140,7 @@ jobs:
             platform_id: win_amd64
     steps:
       - uses: actions/checkout@v4
+      - uses: lukka/get-cmake@latest
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.22
         env:

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -53,7 +53,7 @@ PYTHON_BINDINGS = "ON"
 RUN_SWIG = "ON"
 BUILD_BY_PIP = "ON"
 # https://github.com/njzjz/openbabel-wheel/issues/49
-CMAKE_POLICY_VERSION_MINIMUM=3.5
+CMAKE_POLICY_VERSION_MINIMUM = "3.5"
 
 [project.urls]
 homepage = "https://github.com/njzjz/openbabel-wheel"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -52,6 +52,8 @@ WITH_INCHI = "ON"
 PYTHON_BINDINGS = "ON"
 RUN_SWIG = "ON"
 BUILD_BY_PIP = "ON"
+# https://github.com/njzjz/openbabel-wheel/issues/49
+CMAKE_POLICY_VERSION_MINIMUM=3.5
 
 [project.urls]
 homepage = "https://github.com/njzjz/openbabel-wheel"

